packages/favfreak/favfreak.install

```diff
@@ -0,0 +1,13 @@
+post_install() {
+  set -e
+  cd /usr/share/favfreak
+  python -m virtualenv venv
```

Comment:
virtualenv is reserved for as the last resort packaging way for complex software with tons or very unmaintained dependencies.

favfreak has only one dependency: [mmh3](https://pypi.org/project/mmh3/) and this dependency is pure python no sub-dependencies, so very easy to package.

Package favfreak with [this template](https://github.com/BlackArch/blackarch-pkgbuilds/blob/master/PKGBUILD-python) and mmh3 with [this template](https://github.com/BlackArch/blackarch-pkgbuilds/blob/master/PKGBUILD-python-lib).

Reply:
Ok Np will go to PKGBUILD only. I was using virtualenv install instructions from github.